armi/bookkeeping/db/databaseInterface.py

```diff
@@ -145,7 +145,12 @@ def interactEveryNode(self, cycle, node):
 
         DBs should receive the state information of the run at each node.
         """
-        # skip writing for last burn step since it will be written at interact EOC
```

Comment:
please copy this comment over

Reply:
Addressed https://github.com/terrapower/armi/pull/1005/commits/661e26a9a2a148719549ff717ce4fd0b745b2ef8